java/midp/com/sun/midp/security/Permissions.java

```diff
@@ -1,1230 +0,0 @@
-/*
```

Comment:
I think we want to keep this file, though you add a customized one under the `java/custom` directory.